base/gmp.jl

```diff
@@ -567,8 +575,8 @@ isneg(x::BigInt) = x.size < 0
 ispos(x::BigInt) = x.size > 0
 
 signbit(x::BigInt) = isneg(x)
-flipsign!(x::BigInt, y::Integer) = (signbit(y) && (x.size = -x.size); x)
-flipsign( x::BigInt, y::Integer) = signbit(y) ? -x : x
+flipsign!(x::BigInt,  y::Integer) = (signbit(y) && (x.size = -x.size); x)
+flipsign( x::BigInt,  y::Integer) = signbit(y) ? -x : x
```

Comment:
Indeed! it comes from that I added a third function in the rebase, so I aligned those two with it, and then deleted this 3rd one as it was not a good idea, and forgot to un-align.